docs/deployment/docker/README.md

```diff
@@ -0,0 +1,36 @@
+# Docker Deployment
+
+This section of the Tetragon documentation will help you get started
+with Tetragon without having to deploy a Kubernetes cluster.
+
+## BTF Requirement
+
+The base kernel should support [BTF](../../../README.md#btf-requirement) or a BTF file should
+be bind mounted on top of `/var/lib/tetragon/btf` inside container.
+
+To check if BTF is enabled on your Linux host system check for the BTF file
+in the standard location:
+
+```
+$ ls /sys/kernel/btf/
+```
+
+## Deployment
+
+To run Tetragon image:
+
+```
+docker run --name tetragon \
+   --rm -it -d --pid=host \
+   --cgroupns=host --privileged \
+   -v /sys/kernel/btf/vmlinux:/var/lib/tetragon/btf \
+   quay.io/cilium/tetragon:latest \
```

Comment:
I don't think we tag latest because it is used by default and we want to avoid that. @michi-covalent should know more.

What I would suggest, instead, is to direct users to the logs of the images we build (both for main builds but also for PRs) and ask them to add this tag.

Reply:
Ok done, both to main and tetragon-ci that @michi-covalent did setup! thank you